src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -30,50 +30,73 @@
 namespace Microsoft.Identity.Client.AppConfig
 {
     /// <summary>
+    /// Sets of configuration parameters that were applied to build the application
```

Comment:
How about "Configuration properties needed to build a public or confidential application object"

Reply:
Done. Thanks @bgavrilMS